__tests__/app.test.js

```diff
@@ -127,4 +127,34 @@ describe("app", () => {
         });
     });
   });
```

Comment:
Cant actually comment on your PATCH tests above, as they're not on this pull request, but you're missing some tests for that. 
- TEST - What if the wrong data value is passed in with inc_votes - i.e. a string. 
- TEST - What is the key is spelt wrong?
- TEST - extra keys on the object?
-